core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -146,3 +148,9 @@ case object KAFKA_0_10_2_IV0 extends ApiVersion {
   val messageFormatVersion: Byte = Record.MAGIC_VALUE_V1
   val id: Int = 9
 }
+
+case object KAFKA_0_10_2_IV1 extends ApiVersion {
```

Comment:
Talked to @ijuma about this, and I don't think we need to bump the internal version number since the brokers do not use offset fetches themselves.